### ISSUE TITLE
petitions.whitehouse.gov no longer functional

### ISSUE THREAD

**Issue by dxgldotorg** [WhiteHouse/petitions#120]
It appears the website petitions.whitehouse.gov has been shut down, possibly by order of the new President.  Please consider re-opening it.

I have forked the source code for safekeeping.

**Comment by ghost** [comment]
@WilliamFeely This is historical material “frozen in time”. The website is no longer updated and links to external websites and some internal pages may not work.

**Comment by dxgldotorg** [comment]
Yes, but why was it shut down?  Will we no longer be allowed to petition the White House?

**Comment by ghost** [comment]
@WilliamFeely The site will be moved to the National Archives along with the content hence its shutdown as it's in a transition period from the current President to the President Elect.

**Comment by dxgldotorg** [comment]
Will the new Administration launch a new petitions site, or is this the end?

**Comment by ghost** [comment]
I have no idea

**Comment by dxgldotorg** [comment]
Looks like this issue can be closed as fixed now; the petitions site has been wiped and re-launched.

**Comment by ohammersmith** [comment]
It's no longer functional again. 

Petitions seem to be not counting votes. Including ones that were only five days ago. 

I'm assuming anyone who could answer this can't respond to the comments publically, without incurring wrath. 

Feel free to contact me directly. I'd love to see any patches that may or may not have been applied in the last few days.

**Comment by dxgldotorg** [comment]
Hi.  This issue is not the one I originally had reported.
However, at least during the Obama administration, I had observed that sometimes vote counts will temporarily stick due to server and/or CDN side caching.

**Comment by ohammersmith** [comment]
It's been "stuck" for a couple days now. I'll double check it's not just a caching issue when I'm at my desktop. 

http://www.independent.co.uk/arts-entertainment/president-donald-trump-arts-funding-nea-neh-cpb-prs-npr-petition-white-house-a7541631.html

**Comment by ohammersmith** [comment]
I hadn't noticed #121 so maybe my question is more appropriate there.
